TeachingRecordSystem/tests/TeachingRecordSystem.TestCommon/TestData.UpdateQualification.cs

```diff
@@ -0,0 +1,74 @@
+using Microsoft.Xrm.Sdk.Messages;
+using TeachingRecordSystem.Core.Dqt.Models;
+
+namespace TeachingRecordSystem.TestCommon;
+
+public partial class TestData
+{
+    public Task UpdateQualification(Action<UpdateQualificationBuilder>? configure)
```

Comment:
Can we make this `DeleteMandatoryQualification(Guid qualificationId, ..., bool? syncEnabled = null)` instead (without a builder)?

Reply:
Done